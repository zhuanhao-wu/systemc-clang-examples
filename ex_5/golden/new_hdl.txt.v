module top_0(
input wire clk,
input wire integer in_port,
output reg integer out_port 
);
sc_event aev;
integer v;
integer w;
integer x;
integer y;


endmodule // top_0
module top_1(
input wire clk,
input wire integer in_port,
output reg integer out_port 
);
sc_event aev;
integer v;
integer w;
integer x;
integer y;


endmodule // top_1