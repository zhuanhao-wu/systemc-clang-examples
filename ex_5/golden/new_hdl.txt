  [
    (hModule top_0)
    [
      (hPortin clk)
      [
        (hType sc_in)
      ]
      (hPortin in_port)
      [
        (hType sc_in)
        (hType int)
      ]
      (hPortout out_port)
      [
        (hType sc_out)
        (hType int)
      ]
      (hVardecl aev)
      [
        (hType sc_event)
      ]
      (hVardecl v)
      [
        (hType int)
      ]
      (hVardecl w)
      [
        (hType int)
      ]
      (hVardecl x)
      [
        (hType int)
      ]
      (hVardecl y)
      [
        (hType int)
      ]
    ]
    [
    ]
  ]
  [
    (hModule top_1)
    [
      (hPortin clk)
      [
        (hType sc_in)
      ]
      (hPortin in_port)
      [
        (hType sc_in)
        (hType int)
      ]
      (hPortout out_port)
      [
        (hType sc_out)
        (hType int)
      ]
      (hVardecl aev)
      [
        (hType sc_event)
      ]
      (hVardecl v)
      [
        (hType int)
      ]
      (hVardecl w)
      [
        (hType int)
      ]
      (hVardecl x)
      [
        (hType int)
      ]
      (hVardecl y)
      [
        (hType int)
      ]
    ]
    [
    ]
  ]
